src/browserServerImpl.ts

```diff
@@ -46,7 +47,7 @@ export class BrowserServerLauncherImpl implements BrowserServerLauncher {
       ignoreDefaultArgs: Array.isArray(options.ignoreDefaultArgs) ? options.ignoreDefaultArgs : undefined,
       ignoreAllDefaultArgs: !!options.ignoreDefaultArgs && !Array.isArray(options.ignoreDefaultArgs),
       env: options.env ? envObjectToArray(options.env) : undefined,
-    });
+    }, toProtocolLogger(options.logger));
     return new BrowserServerImpl(browser, options.port);
```

Comment:
I believe it would be helpful to also see logs of incoming connections and calls of the methods on BrowserServer (there was a question about the connection logging in the bug).